spiketoolkit/comparison/sortingcomparison.py

```diff
@@ -342,35 +349,32 @@ def _do_counting(self, verbose=False):
                 lab_st2 = self._labels_st2[self.getMappedSorting1().getMappedUnitIds(u1)]
                 mapped_st = self.getMappedSorting1().getUnitSpikeTrain(u1)
                 # from gtst: TP, TPO, TPSO, FN, FNO, FNSO
-                for sp_i, n_sp in enumerate(sorting1.getUnitSpikeTrain(u1)):
-                    id_sp = np.where((mapped_st > n_sp - self._delta_tp) & (mapped_st < n_sp + self._delta_tp))[0]
-                    if len(id_sp) == 1:
+                for sp_i, n_sp in enumerate(sts1[u_i]):
+                    matches = (np.abs(mapped_st.astype(int)-n_sp)<=self._delta_tp//2)
```

Comment:
With `(np.abs(mapped_st.astype(int)-n_sp)<=self._delta_tp//2)` it will b the closest one. TheMEArec data did have spikes within 20 frames in ground truth (`delta_tp` in the old implementation actually checked 20 frames, or rather 19 I think). With high sampling and high noise, I think 10 frames (5 back, 5 forward) is maybe too low to catch all matches, so I suggest we leave it as it is.

Reply:
right! I see it now :)